File: src/main/java/com/test/functional/java/CallGenerateProcessor.java
package com.test.functional.java;

import com.test.functional.java.enums.PredicateCall;
import com.test.functional.java.model.IncomingCall;

public class CallGenerateProcessor {

    public IncomingCall getCall(int number){
        IncomingCall call = new IncomingCall();
        call.name = "name" + number;
        call.lastname = "lastname" + number;
        call.phone = "8908121683" + number;
        return call;
    }
}

File: src/main/java/com/test/functional/kotlin/CallGenerateProcessor.kt
package com.test.functional.kotlin

import com.test.functional.kotlin.model.IncomingCall

class CallGenerateProcessor {

    fun getCall(number: Int): IncomingCall {
        return IncomingCall(
            name = "name$number",
            lastname = "lastname$number",
            phone = "8908121683$number"
        )
    }
}